### PR TITLE
Event Creator Dashboard

### PR DESCRIPTION
**Pull request by ZAINABFATIMA0**
Added functionality that a logged in user can:
  - Get all events he created
  - The number of registered users and their emails in a particular event
  - The number of available seats in that event

### CODE REVIEW

**Review by Hamza-Chaudhary65 — APPROVED** [review]

File: users/serializer.py [hunk 1]
```diff
@@ -61,3 +61,9 @@ def create(self, validated_data):
         )
         send_otp_email.delay(registration.email, registration.otp, config.OTP_EXPIRY_TIME)
         return registration
+
+
+class VerifiedEmailSerializer(serializers.ModelSerializer):
```
Comment (sheralim012) [inline users/serializer.py]: name isn't matching use and the model being used

File: users/views.py [hunk 2]
```diff
@@ -13,3 +18,34 @@ def register(request):
...
+   serializer = EventSerializer(events_page, many=True)
+   response = paginator.get_paginated_response(serializer.data)
+  
+   return response
+
+@api_view(['GET'])
+@permission_classes([IsAuthenticated])
+def get_verified_registrations(request, pk):
+   
+   event = get_object_or_404(Event, id=pk)
```
Comment (sheralim012) [inline users/views.py]: you could add user check here

**Review by sheralim012 — CHANGES_REQUESTED** [review]

File: users/views.py [hunk 3]
```diff
@@ -13,3 +18,31 @@ def register(request):
...
+   
+   event = get_object_or_404(Event, id=pk, creator=request.user)
+  
+   verified_registrations = Registration.objects.filter(event=event, is_verified=True)
+   serializer = VerifiedRegistrationsSerializer(verified_registrations, many=True)
+  
+   return Response({
+        'seats left': event.seat_limit - verified_registrations.count(),
+        'registrations count': verified_registrations.count(),
+        'registered emails': serializer.data,
```
Comment (sheralim012) [inline users/views.py]: read about recommended api responses
usually we use underscores in keys

**Review by sheralim012 — CHANGES_REQUESTED** [review]

File: users/views.py (hunk 3 above)
Comment (ZAINABFATIMA0) [reply users/views.py]: I'll read about this tonight. For now i have updated this accordingly.

**Review by sheralim012 — APPROVED** [review]

**Pull request merged**